httpd-ldapauth-proxy/templates/config.yaml

```diff
@@ -197,24 +198,49 @@ data:
     SSLRandomSeed startup builtin
     SSLRandomSeed connect builtin
     </IfModule>
+    Include /tmp/httpdldap.conf
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: entrypoint
+data:
+  entrypoint.sh: |-
+    #!/bin/bash
+    export CONFIGDIR=/run/config/
+    cp ${CONFIGDIR}httpdldaptemplate.conf /tmp/httpdldap.conf
+    sed -i  "s/__BINDDN__/$(cat /run/secrets/camptocamp/ldap/login)/g" /tmp/httpdldap.conf 
+    sed -i  "s/__BINDDNPASSWORD__/$(cat /run/secrets/camptocamp/ldap/password)/g" /tmp/httpdldap.conf 
```

Comment:
What happens if the volume is not mounted?

Wouldn't it be easier to use environment variables from secret ?

Reply:
no, i dont want to use env variables in secret.